airflow/kubernetes/pod_launcher.py

```diff
@@ -143,17 +147,88 @@ def base_container_is_running(self, pod):
         wait=tenacity.wait_exponential(),
         reraise=True
     )
-    def read_pod_logs(self, pod):
+    def _read_pod_log_chunk(self, pod: Pod, last_line: bytes) -> HTTPResponse:
+        # The CoreV1Api doesn't support since_time even though the API does, so we must use
```

Comment:
Laaame :( Is this still in place? And is it just a limitation of the python libraries rather than the kube API?